Detectors/ZDC/workflow/src/ZDCDataReaderDPLSpec.cxx

```diff
@@ -33,6 +33,21 @@ void ZDCDataReaderDPLSpec::init(InitContext& ic)
 
 void ZDCDataReaderDPLSpec::run(ProcessingContext& pc)
 {
+  // if we see requested data type input with 0xDEADBEEF subspec and 0 payload this means that the "delayed message"
+  // mechanism created it in absence of real data from upstream. Processor should send empty output to not block the workflow
+  {
+    std::vector<InputSpec> dummy{InputSpec{"dummy", ConcreteDataMatcher{o2::header::gDataOriginFT0, o2::header::gDataDescriptionRawData, 0xDEADBEEF}}};
```

Comment:
use ZDC rather than FT0

Reply:
This was already fixed. Don't understand why it appears to be fixed here..